DFC.ServiceTaxonomy.Editor/wwwroot/tabs.css

```diff
@@ -0,0 +1,9 @@
+﻿.nav-tabs .nav-item .nav-link {
+    color: #007bff !important;
+    background: none !important;
+}
+
+    .nav-tabs .nav-item .nav-link.active {
+        color: #212529 !important;
+        background: white !important;
+    }
```

Comment:
This file should really be part of the DFC.ServiceTaxonomy.CustomFields project, so that the functionality is self-contained.